graphgym/main.py

```diff
@@ -41,20 +41,21 @@
         seed_everything(cfg.seed)
         auto_select_device()
         # Set machine learning pipeline
-        loaders = create_loader()
-        loggers = create_logger()
+        datamodule = GraphGymDataModule()
         model = create_model()
-        optimizer = create_optimizer(model.parameters(), cfg.optim)
-        scheduler = create_scheduler(optimizer, cfg.optim)
         # Print model info
         logging.info(model)
         logging.info(cfg)
         cfg.params = params_count(model)
         logging.info('Num parameters: %s', cfg.params)
         # Start training
         if cfg.train.mode == 'standard':
-            train(loggers, loaders, model, optimizer, scheduler)
+            train(model, datamodule, logger=True)
         else:
+            loaders = create_loader()
+            loggers = create_logger()
+            optimizer = create_optimizer(model.parameters(), cfg.optim)
+            scheduler = create_scheduler(optimizer, cfg.optim)
             train_dict[cfg.train.mode](loggers, loaders, model, optimizer,
```

Comment:
do we need this else block? What is the other `train.mode`?

Reply:
seems like its just 'standard' or "not" 'standard'

Reply:
I think this can be safely dropped.